### PR TITLE
week 3 progress -- fragments and a whole lotta XML!

### PR DESCRIPTION
**Pull request by robert-chatterton**
check out these new view fragments!!
wk3-1
wk3-3
wk3-4

### CODE REVIEW

File: app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package jayp.bge.playboikotlin.robbie.fragments
+
+import android.os.Bundle
```
Comment (zohebaziz) [inline app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt:3]: bundle exec

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -9,6 +9,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.PlayboiKotlin">
+        <activity
+            android:name=".robbie.fragments.RobbieFragmentsActivity"
```
Comment (pelusodan) [inline app/src/main/AndroidManifest.xml:13]: beautiful!

File: app/src/main/res/drawable-anydpi/ic_leaf.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (zohebaziz) [inline app/src/main/res/drawable-anydpi/ic_leaf.xml:1]: this different

File: app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.setupWithNavController
+import jayp.bge.playboikotlin.R
+import jayp.bge.playboikotlin.databinding.ActivityRobbieFragmentsBinding
+
+class RobbieFragmentsActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityRobbieFragmentsBinding
+    private lateinit var navController: NavController
```
Comment (pelusodan) [inline app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt:15]: we should cut this lateinit bullshit out. :/

File: app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+        val navHostFragment = supportFragmentManager.findFragmentById(
+            R.id.robbie_nav_host_container
+        ) as NavHostFragment
+        navController = navHostFragment.navController
+
+        // Setup the bottom navigation view with navController
+        binding.robbieBottomNav.setupWithNavController(navController)
+
+//        setupActionBarWithNavController(navController, appBarConfiguration)
```
Comment (pelusodan) [inline app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt:31]:
```suggestion
//remove this fucking line
```

File: app/src/main/res/layout/fragment_robbie_list.xml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        android:autofillHints="type something here"
+        android:inputType="text"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/robbie_list_button"
+        android:layout_width="300dp"
```
Comment (pelusodan) [inline app/src/main/res/layout/fragment_robbie_list.xml:20]: please add a margin to this.

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -13,6 +13,11 @@
     <string name="title_web_loader">Web Loader</string>
     <string name="title_list_o_rama">List-o-rama</string>
     <string name="title_chatty">Chatty</string>
-<!-- TODO: Remove or change this placeholder text -->
+    <!-- TODO: Remove or change this placeholder text -->
```
Comment (pelusodan) [inline app/src/main/res/values/strings.xml:16]: untab this line you monster

**Review by pelusodan — COMMENTED** [review]
couple nit comments - would appreciate if you fixed 🙁

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -13,6 +13,11 @@
     <string name="title_web_loader">Web Loader</string>
     <string name="title_list_o_rama">List-o-rama</string>
     <string name="title_chatty">Chatty</string>
-<!-- TODO: Remove or change this placeholder text -->
```
Comment (zohebaziz) [inline app/src/main/res/values/strings.xml:16]: good call on this remove

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -13,6 +13,11 @@
...
+    <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (zohebaziz) [inline app/src/main/res/values/strings.xml:17]: hey

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -13,6 +13,11 @@
...
+    <string name="title_robbie_web_loader">Robbie Web Loader</string>
+    <string name="title_robbie_list">Robbie List</string>
+    <string name="web_loader">Web Loader</string>
+    <string name="list_button">List Button</string>
+    <string name="sub_fragment">sub fragment</string>
```
Comment (zohebaziz) [inline app/src/main/res/values/strings.xml:22]: sub

**Review by zohebaziz — APPROVED** [review]

File: app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt (hunk 4 above)
Comment (robert-chatterton) [reply app/src/main/java/jayp/bge/playboikotlin/robbie/fragments/RobbieFragmentsActivity.kt:15]: why should we cut this

**Comment by robert-chatterton** [comment]
I'm merging this man

**Pull request merged**